src/main/java/Group15/View/WorkoutView.java

```diff
@@ -169,6 +189,7 @@ private static Pane createButtonPane(Workout workout){
                     case "Edit Workout" -> ViewController.setScene(EditWorkoutView.createScene(WorkoutView.workout));
                     case "Save" -> {
                         try {
+                            createSuccessSaveDialog();
```

Comment:
You should probably create the success dialog after trying to save

Reply:
The workout to pdf doesnt seem to work, but i have changed it to createDialog after its use